mozetl/graphics/graphics_telemetry_dashboard.py

```diff
@@ -96,8 +94,6 @@ def parse_args():
     MonitorsKey,
     ArchKey,
     DeviceResetReasonKey,
-    SANITY_TEST,
-    SANITY_TEST_REASON,
```

Comment:
We should keep `SANITY_TEST` since it doesn't expire (https://probes.telemetry.mozilla.org/?search=GRAPHICS_SANITY_TEST) and it's still used in the dashboard.  I think we should first try just removing things related to `SANITY_TEST_REASON` which is just the constants.